lib/transform.js

```diff
@@ -47,25 +45,16 @@ module.exports = class Transform {
      * @returns {Map} Map with keys as slot attribute name and corresponding values consisting of array of matching nodes
      */
     _groupSlots(root) {
-        const slotMap = new Map();
-        slotMap.set('default', []);
-        const nodes = adapter.getChildNodes(root);
-        nodes.forEach((node) => {
-            if (adapter.isTextNode(node)) {
-                return;
-            }
-            const attribs = node.attribs;
-            if (attribs && attribs.slot) {
-                if (slotMap.has(attribs.slot)) {
-                    slotMap.get(attribs.slot).push(node);
-                } else {
-                    slotMap.set(attribs.slot, [node]);
-                }
-                this._pushText(node.next, slotMap.get(attribs.slot));
-                delete attribs.slot;
-            } else {
-                slotMap.get('default').push(node);
-                this._pushText(node.next, slotMap.get('default'));
+        const slotMap = new Map([['default', []]]);
+        const nodes = treeAdapter.getChildNodes(root);
+        nodes.forEach(node => {
+            if (!treeAdapter.isTextNode(node)) {
+                const { slot = 'default' } = node.attribs;
+                const slotNodes = slotMap.get(slot) || [];
+                const updatedSlotNodes = [...slotNodes, node];
+                slotMap.set(slot, updatedSlotNodes);
+                this._pushText(node.next, updatedSlotNodes);
```

Comment:
@diogodadalt No issues man.. I just came to know there should be a test for this as well.. Will add it after this PR has landed.